src/dbnode/server/server.go

```diff
@@ -291,6 +293,11 @@ func Run(runOpts RunOptions) {
 
 	opentracing.SetGlobalTracer(tracer)
 
+	debugWriter, err := xdebug.NewZipWriterWithDefaultSources(
+		cpuProfileDuration,
+		iopts,
+	)
+
```

Comment:
A little confused why you dont check the error here, but then you check it later below? Seems like you should just check it here (and fatal out or whatever we do for errors like this in `server.go` and then not check it in the goroutine below

Reply:
Yeah that was dumb, forgot to pull up the error handling. Should be fixed now